docs/models/index.md

```diff
@@ -1,16 +1,61 @@
+    - [Virtual classroom](https://profiles.adlnet.gov/profile/4719f43e-28ef-4108-b76a-5fbde91c6f68)
+
+2. **OpenEdx statements models**:
+    - [Enrollment](https://edx.readthedocs.io/projects/devdata/en/latest/internal_data_formats/tracking_logs/student_event_types.html#enrollment-events)
+    - [Navigational](https://edx.readthedocs.io/projects/devdata/en/latest/internal_data_formats/tracking_logs/student_event_types.html#course-navigation-events)
+    - [Open Reponse Assessment](https://edx.readthedocs.io/projects/devdata/en/latest/internal_data_formats/tracking_logs/student_event_types.html#open-response-assessment-events)
+    - [Peer instruction](https://edx.readthedocs.io/projects/devdata/en/latest/internal_data_formats/tracking_logs/student_event_types.html#peer-instruction-events)
+    - [Problem interaction](https://edx.readthedocs.io/projects/devdata/en/latest/internal_data_formats/tracking_logs/student_event_types.html#problem-interaction-events)
+    - [Textbook interaction](https://edx.readthedocs.io/projects/devdata/en/latest/internal_data_formats/tracking_logs/student_event_types.html#textbook-interaction-events)
+    - [Video interaction](https://edx.readthedocs.io/projects/devdata/en/latest/internal_data_formats/tracking_logs/student_event_types.html#video-interaction-events)
+
+## Statements validation
+
+In learning analytics, the validation of statements takes on significant importance. 
+These statements, originating from diverse sources, systems or applications, must align with specific standards such as [xAPI](https://xapi.com/) for the best known. 
+The validation process becomes essential in ensuring that these statements meet the required standards, facilitating data quality and reliability. 
+
+Ralph allows you to automate the validation process in your production stack. 
+OpenEdx related events and xAPI statements are supported.
+
+!!! warning 
+
+    For now, validation is effective only with supported [learning statement models](#learning-statement-models) on Ralph. About xAPI statements, an [issue](https://github.com/openfun/ralph/issues/388) is open to extend validation to any xAPI statement.
+
+Check out tutorials to test the validation feature:
+
+- [`validate` with Ralph as a CLI](../tutorials/cli.md#validate-command)
+- [`validate` with Ralph as a library](../tutorials/library.md#validate-method)
+
+## Statements conversion
+
+Ralph currently supports conversion from OpenEdx learning events to xAPI statements. Here is the up-to-date conversion sets availables: 
+
+| FROM | TO |
+|---|---|
+|[edx.course.enrollment.activated](https://edx.readthedocs.io/projects/devdata/en/latest/internal_data_formats/tracking_logs/student_event_types.html#edx-course-enrollment-activated-and-edx-course-enrollment-deactivated)|[registered to a course](https://profiles.adlnet.gov/profile/d0f8a66c-df56-4cb4-bfaf-10c2febd87a1/templates/685a9da1-0d8c-4ae5-82bf-fff7b601c288)|
```

Comment:
What about using reference-style markdown links here? (as we do in the changelog for example)